sds_data_manager/lambda_code/SDSCode/downstream_dependency_config.py

```diff
@@ -0,0 +1,164 @@
+"""Stores the downstream dependency configuration of the instruments.
```

Comment:
I wonder if it makes sense to combine this file with downstream file. I kept them separate for now for easy tracking and updating. But open for suggestions.

Reply:
Probably it is better to eventually have them combined. For example if they are separate it would be easier for them to not match (one gets updated and the other one forgotten).